inst/doc/custom_data_sources.Rmd

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
So, I think...yeah, if we weren't building GRAN directly from this repo, we could generally take out the files in inst/doc until we submitted to CRAN. But, since we're building it directly from this (without building vignettes in the GRAN build...we *did* try building vignettes, but had a lot of problems, so decided to generally follow CRAN's lead). Same reason we need the files in the `man` folder.